pkg/grpc/api.go

```diff
@@ -64,6 +64,9 @@ type api struct {
 	appChannel            channel.AppChannel
 	stateStores           map[string]state.Store
 	secretStores          map[string]secretstores.SecretStore
+	defaultSecretAccess   map[string]string
+	allowedSecrets        map[string]map[string]struct{}
+	deniedSecrets         map[string]map[string]struct{}
```

Comment:
if we have a struct for each store's policy these three can be `map[string]StorePolicy` (please choose a name that you like, I am terrible with naming)

Reply:
same here, going with `ParsedSecretsConfiguration` for now